src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposableFieldsShouldBeDisposedTests.cs

```diff
@@ -385,7 +383,7 @@ End Sub
         [Fact, WorkItem(3042, "https://github.com/dotnet/roslyn-analyzers/issues/3042")]
         public async Task AsyncDisposableAllocationInConstructor_AssignedDirectly_Disposed_NoDiagnostic()
         {
-            await VerifyCS.VerifyAnalyzerAsync(IAsyncDisposable.CSharp + @"
```

Comment:
`IAsyncDisposable` is now an available type so I have to use the full namespace for disambiguity.

Reply:
Prior to the TFM change, we should remove the MinimalImplementations.IAsyncDisposable class and add a NuGet reference to Microsoft.Bcl.AsyncInterfaces instead.

Reply:
See #3276

Reply:
I won't have time tonight but will be able to work on it tomorrow.